Bottom: 05a0eb32edc286b12f857deab71e245408f68cb0
Top:    40c997c7a416141f18f67a6a0e3f46260212efab
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-25 01:54:10 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 01d3f46..bd2be04 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -17,15 +17,9 @@
 #include "crosshair.h"
 #include "error.h"
 #include "../hidint.h"
+#include "gui.h"
 #include "hid/common/draw_helpers.h"
 
-#ifdef ENABLE_GL
-#  include <gtk/gtkgl.h>
-#  include <GL/gl.h>
-#  include "hid/common/hidgl.h"
-#endif
-
-#include "gui.h"
 
 #if !GTK_CHECK_VERSION(2,8,0) && defined(HAVE_GDK_GDKX_H)
 #include <gdk/gdkx.h>
@@ -325,10 +319,11 @@ ghid_set_layer (const char *name, int group, int empty)
 	     && group <
 	     max_layer) ? PCB->LayerGroups.Entries[group][0] : group;
 
-  if (idx >= 0 && idx < max_layer + 2) {
-    gport->trans_lines = TRUE;
-    return /*pinout ? 1 : */ PCB->Data->Layer[idx].On;
-  }
+  if (idx >= 0 && idx < max_layer + 2)
+    {
+      gport->trans_lines = TRUE;
+      return /*pinout ? 1 : */ PCB->Data->Layer[idx].On;
+    }
   if (idx < 0)
     {
       switch (SL_TYPE (idx))
